maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -31105,7 +31105,9 @@
 	pixel_x = 5;
 	pixel_y = 29
 	},
-/obj/structure/machinery/chem_master,
+/obj/structure/machinery/chem_master{
```

Comment:
Spoke to nanu about this, rather than instanced in the map editor make a subtype e.g. `/obj/structure/machinery/chem_master/vial` with it's `vial_maker` set to `TRUE`.